charts/devlake/values.yaml

```diff
@@ -182,6 +207,25 @@ ui:
     useSecret: false
     autoCreateSecret: true
     secretName: devlake-auth
+  
+  extraLabels: {}
+
+  ## SecurityContext holds pod-level security attributes and common container settings.
+  ## This defaults to non root user with uid 101 and gid 1000. *v1.PodSecurityContext  false
```

Comment:
Shall we add more context here? For example, stating that the Nginx base image uses uid:gid 101:101, so the uid/fsgroup must be 101 and can't be changed, otherwise config-ui won't be able to overwrite the default config file.